### PR TITLE
Add a PHP Getting Started page

### LINKED ISSUE #240: Missing Getting Started with PHP

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #240

### CODE REVIEW

File: _posts/getting-started/php/2018-02-23-getting-started-with-php.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
+---
+title: Getting Started with PHP
+modified_at: 2018-02-23 00:00:00
+category: getting-started
+tags: follow-the-light php
+---
+
+This tutorial will help you deploying a Go application in a few minutes. We will
```
Comment (yannski) [inline _posts/getting-started/php/2018-02-23-getting-started-with-php.md]: s/Go/PHP/

File: _posts/getting-started/php/2018-02-23-getting-started-with-php.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+the sample:
+
+```bash
+$ git clone https://github.com/Scalingo/sample-php-base.git
+$ cd sample-php-base
+```
+
+You now have locally a functional Git repository that contains a simple PHP
+application. Note the presence of a `composer.json` file at the root of the
+project. Scalingo uses Composer for dependency management in PHP application.
```
Comment (yannski) [inline _posts/getting-started/php/2018-02-23-getting-started-with-php.md]: s/ in PHP application// ?

File: _posts/getting-started/php/2018-02-23-getting-started-with-php.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+%}) guide to setup Git and SSH
+- installed the [command line interface tool]({% post_url
+cli/2015-09-18-command-line-tool %})
+
+## Deploy an Application
+
+### Prepare the Application Code
+
+Let's start by getting an sample application code. First clone the repository of
+the sample:
```
Comment (yannski) [inline _posts/getting-started/php/2018-02-23-getting-started-with-php.md]: why the sample? Why not a simple Hello World app?

**Comment by Soulou** [comment]
@EtienneM it should link to the buildpack configuration/composer.json configuration page

**Comment by EtienneM** [comment]
This page? https://doc.scalingo.com/languages/php/start#top-of-page

**Pull request merged**